release-notes-1.1.0.md

```diff
@@ -4,5 +4,10 @@
 ${version-number}
 
 #### New Features
+- None
+
+#### Patch Fixes Included
+- US517060 - Flyway version upgraded to [9.3.1](https://flywaydb.org/documentation/learnmore/releaseNotes#9.3.1)
```

Comment:
This upgrade feels more like a new feature than a patch fix to me.

Reply:
I have updated the release note and put this update under new feature.